ui.frontend/src/view/FormFileInputWidgetBase.js

```diff
@@ -0,0 +1,484 @@
+/*******************************************************************************
```

Comment:
Why have we created base and widget as two separate classes

Reply:
Base Widget class is common class for all versions (V1, V2,V3), whereas widget class consists of methods particular to V-2 (Drag & Drop Specifically), V3 also requires the same functionality hence we extracted the common methods to super base class. This is only done to avoid code duplication.